### ISSUE TITLE
[ENHANCEMENT] supress matplotlib etc DEBUG info in dh5view (like VisGUI already does)

### ISSUE THREAD

**Issue by csoeller** [python-microscopy/python-microscopy#650]
**Is your feature request related to a problem? Please describe.**

On both macOS and win10 py3 installs of PYME dh5view produces huge numbers of PYME unrelated debug statements, see screenshot:

![pyme-py3-dh5view-debug-msgs](https://user-images.githubusercontent.com/3750860/100646824-7a6ac400-3336-11eb-9f60-1586a2882c05.png)

**Describe the solution you'd like**
VisGUI has the following statements to suppress this:

```
logging.getLogger('matplotlib.font_manager').setLevel(logging.ERROR) #clobber unhelpful matplotlib debug messages
logging.getLogger('matplotlib.backends.backend_wx').setLevel(logging.ERROR)
logging.getLogger('PIL.PngImagePlugin').setLevel(logging.ERROR)
```

**Describe alternatives you've considered**
NA

**Are you prepared to work on it?**
Sure, some guidance where to insert the relevant statements would be great.

**Comment by David-Baddeley** [comment]
Should be as simple as copying lines 57-61 of VisGUI.py to dsviewer.py (around line 48). The only caveat is to make sure that matplotlib and PIL have been imported before you set the log level on their loggers.

**Comment by David-Baddeley** [comment]
Note: copying line 57 is not necessary for this to work, but nice anyway - means that log messages from dsviewer get tagged with the module name rather than, I think `MAIN`.

**Comment by csoeller** [comment]
Cool, will raise a PR soonish...

**Comment by David-Baddeley** [comment]
should be fixed with #651
